igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/JenkinsConfig.groovy

```diff
@@ -40,11 +43,14 @@ import javax.validation.Valid
 @CompileStatic
 class JenkinsConfig {
 
+    @Value('${client.timeout:30}')
```

Comment:
is clientTimeout seconds or millis? I'd have expected millis and if so 30 is going to be not good as a default